src/index.js

```diff
@@ -6,7 +6,8 @@ import Guest from './Guest';
 import Manager from './Manager'
 import Bookings from './Bookings';
 
-let currentUser, bookings, userBookings, selectedDate, guestNames;
+
+let currentUser, manager, bookings, userBookings, selectedDate, guestNames;
```

Comment:
I like how you strung all the let variables together on one line. This is much cleaner looking.